### PR TITLE
For new versions of SVT

### PR DESCRIPTION
**Pull request by linuxgurugamer**
New files, new directory layout

### CODE REVIEW

**Comment by Blu3wolf** [comment]
Closes #5235

**Comment by linuxgurugamer** [comment]
I changed the fixed filenames to regexes, need someone to double-check before merging

File: NetKAN/SVT-MacLinux.netkan [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "spec_version"   : "v1.4",
+    "$kref"          : "#/ckan/github/Galileo88/Stock-Visual-Terrain/asset_match/SVT.*(?i).Mac.Linux.zip",
```
Comment (politas) [inline NetKAN/SVT-MacLinux.netkan]: `"#/ckan/github/Galileo88/Stock-Visual-Terrain/asset_match/SVT.*Mac.Linux.zip"` would suffice and be less confusing.

File: NetKAN/SVT-Windows.netkan [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+    "spec_version"   : "v1.4",
+    "$kref"          : "#/ckan/github/Galileo88/Stock-Visual-Terrain/asset_match/SVT.*(?!Mac).zip",
```
Comment (politas) [inline NetKAN/SVT-Windows.netkan]: `"#/ckan/github/Galileo88/Stock-Visual-Terrain/asset_match/SVT.*(?!Mac).zip$"`  - Without the final "$", it matches both zip file name patterns.

**Comment by linuxgurugamer** [comment]
#rebuild

**Comment by linuxgurugamer** [comment]
@politas Changes were made, can you take a look?
Thx

**Comment by Dazpoet** [comment]
Seems we're grabbing the incorrect .zip somehow. Since I have a firm belief that regexes are 50% logic and 50% magic I won't even try to decipher why, I'll just note that we're not getting what we want to get.

***"download": "https://github.com/Galileo88/Stock-VisualTerrain/releases/download/2.0.2/SVT.v2.0.2.Mac.Linux.zip"***
***"name": "Stock Visual Terrain (for Windows)"***

**Comment by linuxgurugamer** [comment]
Since I stupidly deleted the old repo, I'm going to have to do another PR since I can't edit the currentl.
I do have the regex fixed

File: NetKAN/SVT-Windows.netkan [hunk 3]
```diff
@@ -0,0 +1,26 @@
+{
+    "spec_version"   : "v1.4",
+    "$kref"          : "#/ckan/github/Galileo88/Stock-Visual-Terrain/asset_match/SVT.*(?!Mac).zip$",
```
Comment (politas) [inline NetKAN/SVT-Windows.netkan]: Ok, that wasn't working, but this does (though I don't really like it)

    "$kref"          : "#/ckan/github/Galileo88/Stock-Visual-Terrain/asset_match/SVT.*[^x]\\.zip$",

**Pull request closed without merging**